DMFT/src/main/java/com/gov/rsi/DMFT/controllers/ReportController.java

```diff
@@ -14,51 +21,123 @@
 
 import com.gov.rsi.DMFT.models.Report;
 import com.gov.rsi.DMFT.repositories.ReportRepository;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.PathNotFoundException;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * Implements endpoints for DMER document requests
+ */
 @RestController
-@RequestMapping("/reports")
-public class ReportController {
+@RequestMapping("/queue/dmer")
+public class ReportController extends AbstractController {
+	
+	// The JSONPath expression to find the element containing the driver's license
+	// Note that in the assumed schema, it is within an "item" element in the Questionnaire
+	// resource, rather than in the "Patient" resource
+	private static final String LICENSE_NUMBER_PATH = "$..item[?(@.linkId==\"{0}\")]";
+	// The linkId of the "item" element holding the license number
+	private static final String LICENSE_NUMBER_ELEMENT = "patientDriversLicenseNumber";
+	
+	private static Logger log = LoggerFactory.getLogger(ReportController.class);

```

Comment:
Is there a way to also get the logger through dependency and configure it through a settings file?

Reply:
Logging configuration is performed by the LogFactory class based on the content of the log4j.properties file. Currently just has standard default config of everything to console.